README.md

```diff
@@ -75,6 +75,33 @@ const {
 const api = new JitsiMeetExternalAPI(...);
 setupAlwaysOnTopRender(api);
 ```
+
+#### WiFi Stats
+Provides a function to query for wifi stats on the host computer. Returns information like interface name, addresses, signal quality, noise (not available on all OS). 
+
+**WiFi Stats:**
+
+In the **main** electron process:
+```Javascript
+const {
+    setupWiFiStats
+} = require("jitsi-meet-electron-utils");
+
+// jitsiMeetWindow - The BrowserWindow instance
+// of the window where Jitsi Meet is loaded.
+setupWiFiStats(jitsiMeetWindow);
```

Comment:
Is it accurate that setupWiFiStats also needs to be called on the window itself? I'm assuming, possibly incorrectly, that you copied from the AlwaysOnTop above, but that seems to need to be called with the window and the iframe using two separate apis.